great_expectations/data_context/store/datasource_store.py

```diff
@@ -90,6 +92,19 @@ def deserialize(self, value: Union[dict, DatasourceConfig]) -> DatasourceConfig:
         else:
             return self._schema.loads(value)
 
+    def ge_cloud_response_json_to_object_dict(self, response_json: dict) -> dict:
+        """
+        This method takes full json response from GE cloud and outputs a dict appropriate for
+        deserialization into a GE object
```

Comment:
Why is this an instance method?

It's not actually using anything from the instance or class, but even just making this a `staticmethod` doesn't totally feel right. 

I think maybe this is just another symptom of our lack of a proper [Service Layer](https://martinfowler.com/eaaCatalog/serviceLayer.html) 🤔

Reply:
I agree - just keeping scope small in this PR and following the existing pattern.

Reply:
service layer makes a lot of sense here - I don't have a ton of experience with implementing one in a codebase of this size but could definitely use your knowledge here @Kilo59! Perhaps something for the backlog